src/refspec.c

```diff
@@ -160,6 +160,30 @@ void git_refspec__free(git_refspec *refspec)
 	memset(refspec, 0x0, sizeof(git_refspec));
 }
 
+int git_refspec_parse(git_refspec **refspec, const char *input, int is_fetch)
+{
+	git_refspec *new_refspec;
+	assert(refspec && input);
+
+	new_refspec = git__malloc(sizeof(git_refspec));
+	GITERR_CHECK_ALLOC(new_refspec);
+
+	is_fetch = !!is_fetch;
+	if (git_refspec__parse(new_refspec, input, is_fetch) != 0) {
```

Comment:
You could just pull the `!!is_fetch` into this function. So `if (git_refspec__parse(new_refspec, input, !!is_fetch) != 0)`